build.gradle

```diff
@@ -75,3 +75,7 @@ compileTestKotlin {
     kotlinOptions.jvmTarget = "1.8"
     kotlinOptions.freeCompilerArgs += ["-Xskip-runtime-version-check"]
 }
+
+test {
+    systemProperty "KOTLIN_VERSION", "$kotlin_version"
```

Comment:
Am I correct to assume that this sets an environment variable `KOTLIN_VERSION` to the current value of the Gradle variable `kotlin_version`?

Reply:
Right, to be used here: https://github.com/tschuchortdev/kotlin-compile-testing/pull/42/files/3c55b819e39078b5e7f5c7c752a1c787a71bf257#diff-6efbdba372511a9217d44dd2924076e0

Link: https://docs.gradle.org/current/dsl/org.gradle.api.tasks.testing.Test.html

Reply:
Alright, I'll merge it then.